src/modules/returns/lib/helpers.js

```diff
@@ -237,11 +259,14 @@ const getReturnsViewData = async (request) => {
 
   const documents = await getLicenceNumbers(request, filter);
   const licenceNumbers = documents.map(row => row.system_external_id);
+  const xmlUpload = await isXmlUpload(licenceNumbers);
+  const externalReturns = await isExternalReturns(request);
```

Comment:
Although this works with an `await`, `isExternalReturns` is synchronous so the `await` is not needed